numba_dppy/spirv_generator.py

```diff
@@ -42,61 +42,49 @@ def check_call(*args, **kwargs):
 
 class CmdLine(object):
     def disassemble(self, ipath, opath):
-        check_call(
-            [
-                "spirv-dis",
-                # "--no-indent",
-                # "--no-header",
-                # "--raw-id",
-                # "--offsets",
-                "-o",
-                opath,
-                ipath,
-            ]
-        )
+        flags = []
+        # flags.append("--no-indent")
```

Comment:
Let us get rid of the commented out lines.

Reply:
So I need another way to provide info for which flags are possible.